contracts/modules/RecoveryManager.sol

```diff
@@ -87,18 +83,14 @@ contract RecoveryManager is RelayerModule {
         IModuleRegistry _registry,
         IGuardianStorage _guardianStorage,
         uint256 _recoveryPeriod,
-        uint256 _lockPeriod,
-        uint256 _securityPeriod,
-        uint256 _securityWindow
+        uint256 _lockPeriod
     )
         BaseModule(_registry, _guardianStorage, NAME)
         public
     {
-        require(_lockPeriod >= _recoveryPeriod && _recoveryPeriod >= _securityPeriod + _securityWindow, "RM: insecure security periods");
+        require(_lockPeriod >= _recoveryPeriod, "RM: insecure security periods");
```

Comment:
It makes sense to remove these 2 parameters from the constructor since they are never used and there is no guarantee that they correspond to the values used in the other modules. But we should add a comment on the contract to make sure we have validated that condition before deployment, something along the line of "For the wallet to be secure we must have `_recoveryPeriod >= _securityPeriod + _securityWindow` where `_securityPeriod` and `_securityWindow` are the security parameters of adding/removing guardians and confirming large transfers.".

Reply:
When I was analysing the various security windows I thought about potentially consolidating these into the `BaseModule` in order to ensure consistency. The only disadvantage I could think of is the ability to change these independently on modules which I'm not sure we'll ever want. What do you think?

```
GuardianManager
securityWindow 43200
securityPeriod 86400

TransferManager
securityWindow 43200
securityPeriod 86400

RecoveryManager
recoveryPeriod 129600
lockPeriod       432000

LockManager
lockPeriod 432000
```

Reply:
There has been several requests from users to increase these security periods so we might change them at some point.  One of these things that we discuss periodically... While it is not concrete enough to influence our design, I still think I would prefer to not have these hardcoded in every module and keep some flexibility.

For now I think a clear comment in the contract is sufficient.

Reply:
done